backend/buildeR.Common/DTO/User/Enums/RoleUserDto.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace buildeR.Common.DTO.User.Enums
+{
+    public enum RoleUserDto

```

Comment:
Do not need copy enums. Let's move it to the Common project and use them for DTOs and entities

Reply:
done